data/data/bootstrap/baremetal/files/usr/local/bin/master-bmh-update.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# shellcheck disable=SC1091
+. /usr/local/bin/release-image.sh
+
+export KUBECONFIG=/etc/kubernetes/kubeconfig
```

Comment:
try to use https://github.com/openshift/installer/blob/master/data/data/bootstrap/systemd/units/approve-csr.service#L7